client/src/plus/notifications/notification-card-list-item.tsx

```diff
@@ -0,0 +1,75 @@
+import dayjs from "dayjs";
+import React from "react";
+import { Button } from "../../ui/atoms/button";
+import { DropdownMenu, DropdownMenuWrapper } from "../../ui/molecules/dropdown";
+import { Checkbox } from "../../ui/molecules/notifications-watch-menu/atoms/checkbox";
+
+export default function NotificationCardListItem({
+  toggleSelected,
+  toggleStarred,
+  item,
+  handleDelete,
+}) {
+  const [show, setShow] = React.useState(false);
+
+  return (
+    <li
+      className={`notification-card ${!item.read ? "unread" : ""}`}
+      key={item.id}
+    >
+      <Checkbox
+        name="selected"
+        checked={item.checked}
+        onChange={(e) => toggleSelected(item, e.target.value)}
+      />
+      <Button
+        type="action"
+        extraClasses="notification-card-star"
+        icon={item.starred ? "star-filled" : "star"}
+        onClickHandler={() => toggleStarred(item)}
+      >
+        <span className="visually-hidden">Toggle Starring</span>
+      </Button>
+
+      <a href={item.url}>
+        <div className="notification-card-description">
```

Comment:
Do we _need_ the `div` element? What would happen if we had:

```suggestion
        <a href={item.url} className="notification-card-description">
```